chore: Update service binding docs (#279)
SAP/ai-sdk-js: packages/foundation-models/README.md

## Prerequisites

- [Enable the AI Core service in SAP BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup).
- Configure the project with **Node.js v20 or higher** and **native ESM** support.

> - Ensure a deployed OpenAI model is available in the SAP Generative AI Hub.

Author: Done

>   - Use the [`DeploymentApi`](https://github.com/SAP/ai-sdk-js/blob/main/packages/ai-api/README.md#create-a-deployment) from `@sap-ai-sdk/ai-api` to deploy a model to the SAP Generative AI Hub. For more information, see [this guide](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-generative-ai-model-in-sap-ai-core). Deployment can be set up for each model and model version, as well as a resource group.

Reviewer: ```suggestion
  - Use the [`DeploymentApi`](https://github.com/SAP/ai-sdk-js/blob/main/packages/ai-api/README.md#create-a-deployment) from `@sap-ai-sdk/ai-api` to deploy a model to the SAP Generative AI Hub.
  For more information, see [this guide](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-generative-ai-model-in-sap-ai-core).
```

Author: Updated in all READMEs

  - Once a deployment is complete, access the model via the `deploymentUrl`.

> **Accessing the AI Core Service via the SDK**:

> > The SDK automatically retrieves the `AI Core` service binding from the `VCAP_SERVICES` environment variable, along with the service token needed for authentication.

Reviewer: ```suggestion
> The SDK automatically retrieves the `AI Core` service credentials from the `VCAP_SERVICES` environment variable and resolves the access token needed for authentication.
```

Author: Updated in all READMEs
